### ISSUE TITLE
NFT Display: v0

### ISSUE THREAD

**Issue by corwinharrell** [near/near-wallet#1788]
## Problem
The ecosystem now offers several ways for people to purchase NFTs. Outside of the original marketplace where these NFTs are minted, there is no way for a NEAR user to view their collection of NFTs that live on NEAR across different platforms.

### How did we discover this problem?
With several high-impact opportunities to onboard users to NEAR via NFTs coming up in the next several weeks, team and community members have anecdotally expressed the need to see their NFT collections from within their wallet.

## Scope
- v0 should allow users to view their collection of all the NFTs associated with a particular wallet account.
- Users should be able to then view more minute details about each NFT in their collection, such as:
  - A larger preview
  - The original artist/account with which it was minted
  - Date Created
  - A description (if provided)
  - A link to the NEAR Explorer transaction as a proof of purchase
  - A link to the original file/content of the NFT on IPFS or wherever it may be hosted

## Out of Scope
- v0 will not include the ability to send or exchange NFTs
- v0 will not include a history of transactions involving each NFT, only the date minted and the last purchase

## How will we measure success?
- When asked to share details of a specific NFT in their collection, a NEAR wallet user should be able to navigate to the NFT details page in the wallet, without assistance, in order to source the relevant metadata. (Example: "What is the description of the last NFT you purchased?
- Method of testing: Prototype usability test

## Design & Development Resources
- [FIGMA Document](https://www.figma.com/file/Co5Rx8i9Cx4SxgRZv0MUa7/NFTs?node-id=0%3A1)

**Comment by stefanopepe** [comment]
From my recent conversation with @vgrichina it seems that backend implementation is blocked/slowed down by some NFT players in the ecosystem using different standards and smart contract calls.
Essentially, every NFT platform requires ad-hoc view methods, along with additional work for the indexer.

We may have to cut some corners to deploy the V0 on time, I'll have @vgrichina weight the UX and raise concerns on what may require too long to be implemented

**Comment by vgrichina** [comment]
> From my recent conversation with @vgrichina it seems that backend implementation is blocked/slowed down by some NFT players in the ecosystem using different standards and smart contract calls.

looks like it's not too bad (based on extra research yesterday), like Mintbase does indeed implement reasonable subset of standard
the only problems are:
- they gonna have limitation on how many NFTs one user can own in one NFT contract and still get them displayed
- stuff gonna load slower than optimally because they don't store metadata on chain

I've discovered something that we didn't take into account in design however 
Apparently every NFT contract also has it's own metadata and icon (similar to fungible token contracts). This essentially corresponds to artists deploying separate shop on Mintbase. Or like separate collectible project like Cryptokitties.

Metadata looks like this:
https://web4.near.page/web4/contract/zkquarks.mintbase1.near/nft_metadata

Basically it has `name`, `symbol` and `icon`.

Feel like it makes sense to group individual NFTs under such collections acting as table sections.

Wonder what are your thoughts on this  @corwinharrell?

**Comment by corwinharrell** [comment]
@vgrichina yeah this makes a lot of sense. What I'll do is add a heading to each "collection" on the main Collectibles page using the name and icon of that collection. That will also be reflected on the NFT details page. I'll update the design first thing Monday 👍 Thanks for the heads up

**Comment by corwinharrell** [comment]
@vgrichina I've added an accordion interaction to accommodate displaying NFTs by collection. Here's a quick example but you can also reference the Figma doc.

<img width="723" alt="Screen Shot 2021-06-01 at 5 40 41 PM" src="https://user-images.githubusercontent.com/5934188/120393788-842ead80-c300-11eb-9eff-3f17d5b7e146.png">

**Comment by Patrick1904** [comment]
V0 shipped here: https://github.com/near/near-wallet/pull/1824
